velox/exec/Task.h

```diff
@@ -30,7 +30,7 @@ class PartitionedOutputBufferManager;
 
 class HashJoinBridge;
 class CrossJoinBridge;
-
+enum UpdateBroadcastStatus { SUCCESS, NO_OP, BUFFERS_NOT_FOUND };
```

Comment:
@karteekmurthys Let's discuss offline the design for this API. 

Please, note that enum values should be named as kSuccess, kNoOp, kBuffersNotFound. Check out the https://github.com/facebookincubator/velox/blob/main/CODING_STYLE.md